src/bitcoin-wallet.cpp

```diff
@@ -26,12 +26,15 @@ static void SetupWalletToolArgs()
 
     gArgs.AddArg("-datadir=<dir>", "Specify data directory", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
     gArgs.AddArg("-wallet=<wallet-name>", "Specify wallet name", ArgsManager::ALLOW_ANY | ArgsManager::NETWORK_ONLY, OptionsCategory::OPTIONS);
+    gArgs.AddArg("-dumpfile=<file name>", "When used with 'dump', writes out the records to this file instead of printing to stdout. When used with 'createfromdump', loads the records into a new wallet.", ArgsManager::ALLOW_STRING, OptionsCategory::OPTIONS);
```

Comment:
Is there any reason to support stdout?

I'd find it easier to force this setting for both settings and it would simplify reasoning about arbitrary code execution. IIRC in bash simply catting a file can execute arbitrary commands.

Reply:
Removed the output to stdout.